project.clj

```diff
@@ -16,7 +16,7 @@
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/certificate-authority "0.6.0"]
+                 [puppetlabs/ssl-utils "0.8.0"]
                  [puppetlabs/http-client "0.4.0"]
```

Comment:
`http-client 0.4.1` replaced the certificate-authority dependency with a dependency on `ssl-utils 0.7.0`.  I have a PR up for clj-http-client that would bump this up to `ssl-utils 0.8.0` - https://github.com/puppetlabs/clj-http-client/pull/36.